packages/core/test/test-util/test-services/v4/test-service/TestComplexType.ts

```diff
@@ -102,7 +102,7 @@ export interface TestComplexType {
 /**
  * @deprecated Since v1.6.0. Use [[TestComplexType.build]] instead.
  */
-export function createTestComplexType_1(json: any): TestComplexType {
+export function createTestComplexType(json: any): TestComplexType {
```

Comment:
I guess this change is related to the changes introduced in the edmx file?

Reply:
Yes, there is no createTestComplexType function import, see below.